File: bot/src/main/java/net/slimediamond/atom/telegram/commands/RandomPlayerCommand.java
package net.slimediamond.atom.telegram.commands;

import net.slimediamond.atom.command.telegram.TelegramCommandContext;
import net.slimediamond.atom.command.telegram.TelegramCommandExecutor;
import net.slimediamond.atom.util.minecraftonline.MinecraftOnlineAPI;

import java.io.IOException;
import java.util.concurrent.ThreadLocalRandom;

public class RandomPlayerCommand implements TelegramCommandExecutor {
    public void execute(TelegramCommandContext ctx) throws IOException {
        MinecraftOnlineAPI.getOnlinePlayers().ifPresentOrElse(players -> {
            String randomPlayer = players.get(ThreadLocalRandom.current().nextInt(players.size()));
            ctx.reply("Random online player: " + randomPlayer);
        }, () -> {
            ctx.reply("No players are online or the server is offline.");
        });
    }
}

File: bot/src/main/kotlin/net/slimediamond/atom/commands/minecraftonline/RandomPlayerCommand.kt
package net.slimediamond.atom.commands.minecraftonline

import com.minecraftonline.mcodata.web.MCOWebDataProvider
import net.slimediamond.atom.api.command.CommandNode
import net.slimediamond.atom.api.command.CommandNodeContext
import net.slimediamond.atom.api.command.CommandResult
import net.slimediamond.atom.api.messaging.richText

class RandomPlayerCommand : CommandNode("Get a random player on MCO", "randomplayer", "rp") {

    override suspend fun execute(context: CommandNodeContext): CommandResult {
        val players = MCOWebDataProvider.web().server.onlinePlayers
        if (players.isEmpty()) {
            context.sendMessage("No players are on MCO right now")
            return CommandResult.success
        }
        val player = players.random()
        context.sendMessage {
            append(richText("A random player on MinecraftOnline right now is "))
            append(richText(player.name).bold())
            append(richText("."))
        }
        return CommandResult.success
    }
}